src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/base/ImageComponent.tsx

```diff
@@ -24,6 +24,7 @@ export interface IImagesrc {
   nb: string;
   nn?: string;
   en?: string;
+  [language: string]:string;
```

Comment:
Nice fix, we could perhaps consider removing the `nn` and `en` values?

Reply:
I think it is nice to keep them for documentation and tab completion, just as suggestions for what kind of values the type will hold.